src/Lykke.Service.LoggingAdapter/Helpers/LogLevelHelper.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+using Lykke.Service.LoggingAdapter.Contracts.Log;
+
+namespace Lykke.Service.LoggingAdapter.Helpers
+{
+    public static class LogLevelHelper
+    {
+        public static Microsoft.Extensions.Logging.LogLevel MapToMicrosoftLoglevel(this LogLevelContract logLevel)
+        {
+            switch (logLevel)
+            {
+                case LogLevelContract.Info:
+                    return Microsoft.Extensions.Logging.LogLevel.Information;
+                case LogLevelContract.Error:
+                    return Microsoft.Extensions.Logging.LogLevel.Error;
+                case LogLevelContract.Warning:
+                    return Microsoft.Extensions.Logging.LogLevel.Warning;
+                case LogLevelContract.Monitor:
+                    return Microsoft.Extensions.Logging.LogLevel.Warning;
```

Comment:
This should be not so straightforward. For monitor level you should use ```IHealthNotifier```. So, you should have HN instance for each AppName and use it for the ```LogLevelContract.Monitor``` entries.